cmd/agent/app/reporter/grpc/builder.go

```diff
@@ -39,8 +40,8 @@ type ConnBuilder struct {
 	TLSCA         string
 	TLSServerName string
 
-	notifier discovery.Notifier
-	resolver resolver.Builder
+	subsetSize int
+	notifier   discovery.Notifier
```

Comment:
why not make these two fields public? And drop WithDiscoveryNotifier() function, it doesn't add anything.

Reply:
sg. Was just following our pattern with tchannel builder.